src/chunk.h

```diff
@@ -1713,6 +1713,7 @@ inline bool Chunk::IsParenOpen() const
 {
    return(  Is(CT_PAREN_OPEN)
          || Is(CT_FPAREN_OPEN)
+         || Is(CT_GPAREN_OPEN)
```

Comment:
There may be other places in the code where there is a similar check for open or closed parenthesis, without the use of `IsParenOpen`/`IsParenClose`. Worth doing a search in the code, IMO.

Reply:
I don't understand what you wish. Please make a proposal.

Reply:
Look for example at https://github.com/uncrustify/uncrustify/blob/master/src/indent.cpp#L2549-L2555: here there is a check for paren open, but it does not use `IsParenOpen`. So the new `CT_GPAREN_OPEN` is not checked. By memory, there are sure other places similar to this.
Anyway this could be solved in a different PR if you wish.